app/views/groups/index.html.erb

```diff
@@ -0,0 +1,17 @@
+<div class="splash2">
+  <div class="top-strip2">
+    <%= button_to 'Sign Out', destroy_user_session_path, class:"signout-link", method: :delete %>
+    <h2>CATEGORIES</h2>
+  </div>
+<% @groups.each do |group| %>
+  <div class="group-container">
+    <div class="group-element">
+      <img src="<%= group.icon %>" class="no-underline group-image">
+    </div>
+    <%= link_to group_deals_path(group), class:"group-link group-element" do %>
+      <p><%= group.name %></p>
+    <% end %>
+    <p class="group-element"><%= number_to_currency(group.deals.where(author: current_user).sum(:amount)) %></p>
+  </div>
+<% end %>
+<%= link_to 'Add a New Category', new_group_path, class:'new-category-link'%>
```

Comment:
- [ ] As per the requirement, kindly move your button to the bottom of the page and style it according to the given design. The same thing applies for the 'add a new transaction' button. Please fix it as well: 

yours|given
---|---
![image](https://github.com/ehmaddd/BudgetApp/assets/91486791/370de71b-9f41-4896-9ed6-e9443ecfa8ef)|![image](https://github.com/ehmaddd/BudgetApp/assets/91486791/cf412b57-0ece-4aec-9a35-6f58f155c374)

Reply:
Removed all the errors
![image](https://github.com/ehmaddd/BudgetApp/assets/54537654/ccadb229-2527-4423-8567-0fd56a247497)

Reply:
Button is sticking to the bottom as required :

![image](https://github.com/ehmaddd/BudgetApp/assets/54537654/c2976e0f-aa03-4e99-9533-201a4d296340)